lib/config/fileconf.go

```diff
@@ -2207,10 +2207,15 @@ type WindowsDesktopService struct {
 	// Hosts is a list of static, AD-connected Windows hosts. This gives users
 	// a way to specify AD-connected hosts that won't be found by the filters
 	// specified in `discovery` (or if `discovery` is omitted).
+	// This field is deprecated.
 	Hosts []string `yaml:"hosts,omitempty"`
 	// NonADHosts is a list of standalone Windows hosts that are not
 	// jointed to an Active Directory domain.
+	// This field is deprecated.
 	NonADHosts []string `yaml:"non_ad_hosts,omitempty"`
```

Comment:
```suggestion
	// NonADHosts is a list of standalone Windows hosts that are not
	// jointed to an Active Directory domain.
	//
	// Deprecated: prefer StaticHosts instead.
	NonADHosts []string `yaml:"non_ad_hosts,omitempty"`
```

This specific format (a new paragraph beginning with `Deprecated: `) is the proper way to mark something deprecated.

https://go.dev/blog/godoc